sous-chef/config/rtk.yaml

```diff
@@ -2,4 +2,4 @@
 rtk:
   delete_older_files: true
   path: ./Dropbox/SharedApps/RecetteTek
-  file_pattern: backup*.rtk
+  file_pattern: myrecipebox_*.rtk
```

Comment:
```suggestion
  file_pattern: *.rtk
```

Reply:
Thanks. The pattern needs quotes but I added your suggestion.